paasta_tools/cli/cli.py

```diff
@@ -38,6 +40,33 @@ def error(self, message):
         sys.exit(1)
 
 
+def list_external_commands():
+    p = subprocess.check_output(["/bin/bash", "-c", "compgen -A command paasta-"])
+    lines = p.decode("utf-8").strip().split("\n")
+    return {l.replace("paasta-", "", 1) for l in lines}
+
+
+def calling_external_command():
+    if len(sys.argv) > 1:
+        return sys.argv[1] in list_external_commands()
+    else:
+        return False
+
+
+def get_command_help(command):
+    return f"(run 'paasta {command} -h' for usage)"
+
+
+def external_commands_items():
+    for command in list_external_commands():
+        command_help = get_command_help(command)
+        yield command, command_help
+
+
+def exec_out_to(command, argv):
+    os.execlp(f"paasta-{command}", *argv[1:])
```

Comment:
it's a nit, but passing first element of args *and* args itself seems redundant?

since this function isn't supposed to be generic command calling facility, what about renaming it to `exec_subcommand` and just pass single `args` argument?

Reply:
Got it